File: Examples/Ruby/Workbook/post_workbook_split.rb
require 'aspose_cells_cloud'

class Workbook

  include AsposeCellsCloud
  include AsposeStorageCloud

  def initialize
    #Get App key and App SID from https://cloud.aspose.com
    AsposeApp.app_key_and_sid("APP_KEY", "APP_SID")
    @cells_api = CellsApi.new  
  end

  def upload_file(file_name)
    @storage_api = StorageApi.new 
    response = @storage_api.put_create(file_name, File.open("../data/" << file_name,"r") { |io| io.read } )
  end

  def post_workbook_split
    file_name = "Sample_Test_Book.xls"
    upload_file(file_name)

    response = @cells_api.post_workbook_split(file_name, {format: "png", from: 1, to: 1})
  end

end

workbook = Workbook.new()
puts workbook.post_workbook_split

File: Examples/PHP/Workbook/PostWorkbookSplit.php
<?php
require_once realpath(__DIR__ . '/..') . '/vendor/autoload.php';
require_once realpath(__DIR__ . '/..') . '/Utils.php';

use Aspose\Cells\CellsApi;
use Aspose\Cells\AsposeApp;

class Workbook {

	public $cells;

	public function __construct() {
		AsposeApp::$appSID = Utils::appSID;
		AsposeApp::$apiKey = Utils::apiKey;
		$this->cells = new CellsApi();
	}
	
	public function postWorkbookSplit() {
    	// Upload file to Aspose Cloud Storage
		$fileName = "Sample_Test_Book.xls";
		Utils::uploadFile($fileName);

        $result = $this->cells->PostWorkbookSplit($name=$fileName, $format = "png", $from = 1, $to = 1, $horizontalResolution = null, $verticalResolution = null, $storage = null, $folder = null);
        print_r($result);
	}
}

$workbook = new Workbook();
$workbook->postWorkbookSplit();

?>